compiler/src/iree/compiler/Codegen/Utils/GPUUtils.h

```diff
@@ -16,6 +17,20 @@ namespace iree_compiler {
 static constexpr int32_t kNumGPUDims = 3;
 static constexpr int32_t kWarpSize = 32;
 
+//===----------------------------------------------------------------------===//
+// Target architecture utils
+//===----------------------------------------------------------------------===//
+
+/// Structure to represent target features.
+struct TargetInfo {
```

Comment:
FYI @dcaballe . I dont know if there is anything that can be commoned here with similar things on other backends, but at least raising visibility of it.